### ISSUE TITLE
let's work on iOS for some time?

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#2244]
@sosnovsky @ioanmo226 I've set up milestone https://github.com/FlowCrypt/flowcrypt-ios/milestone/48 to have some useful features in it for iOS. 

I would in particular like Ioan to first do https://github.com/FlowCrypt/flowcrypt-browser/issues/5215 on browser extension before moving to iOS. 

Ideally also the other 3 assigned issues, but not 100% required.

![image](https://github.com/FlowCrypt/flowcrypt-ios/assets/6306961/86269c06-073c-4f64-bc57-a25a3ec64361)

Was there something else one of you wanted to do on browser extension, before Ioan makes the switch?

I'm imagining, Ioan could be focusing on iOS for 2-3 upcoming months if that's ok.

**Comment by tomholub** [comment]
Also, our TestFlight build is expired, which is quite annoying for users who may have used it. What should we do? If we go through dry spells like this in the future, should we just not have a TestFlight public build? Or should we be making builds every 60 days regardless?

**Comment by ioanmo226** [comment]
Nothing for my side.
> Was there something else one of you wanted to do on browser extension, before Ioan makes the switch?

Not sure there is good way but AFAIK apple recommends to update app regularly during beta testing period
> Also, our TestFlight build is expired, which is quite annoying for users who may have used it.

**Comment by sosnovsky** [comment]
> Was there something else one of you wanted to do on browser extension, before Ioan makes the switch?

No, these tasks should be enough before switching to iOS, also https://github.com/FlowCrypt/flowcrypt-browser/issues/5185 doesn't need any work for now - probably this issue was caused by broken signature file, still waiting for customer's response.

> I'm imagining, Ioan could be focusing on iOS for 2-3 upcoming months if that's ok.

Yeah, that's ok, as currently there are no urgent tasks for browser extension, while iOS has useful functionality which needs to be implemented.

> Also, our TestFlight build is expired, which is quite annoying for users who may have used it. What should we do? If we go through dry spells like this in the future, should we just not have a TestFlight public build? Or should we be making builds every 60 days regardless?

I think it's useful to have public build, as we have many users of it and can detect possible issues before releasing to App Store. Previous build expired a month ago, so I think for now there is no reason to re-upload the same build with increased build number, let's upload a new build when some new functionality will be implemented. And then we should monitor that new build is uploaded before previous one is expired (every 90 days)

**Comment by tomholub** [comment]
Hmm, but if we are going to still maintain the TestFlight, then imo we should do a build with increased number (but not increased version) now. It would take yet another month before we can add useful functionality.

**Comment by martgil** [comment]
We haven't received any complaints about our TestFlight public build being unavailable. However, some users may have opted for the App Store version of our FlowCrypt app.
